BurnOutSharp/ProtectionType/SafeDisc.cs

```diff
@@ -51,6 +83,29 @@ public string CheckPortableExecutable(string file, PortableExecutable pex, bool
             if (stxt371Section || stxt774Section)
                 return $"SafeDisc {Get320to4xVersion(null, null, null)}";
 
+            // Get the BSS section, if it exists
+            var BSSSectionRaw = pex.ReadRawSection("BSS", first: true);
+            if (BSSSectionRaw != null)
+            {
+                var matchers = new List<ContentMatchSet>
+                {
+                    // Present in "mcp.dll", along with several other references to Plextor (Redump entries 28810 and 30555).
+                    // plextor.dll.IsPlextor.PreventLowSpeed
```

Comment:
I'm hesitant to have any reference to `plextor.dll` be an indication of SafeDisc or SafeCast. I think this is a valuable piece of information, but I fear that it could overmatch too easily.

Reply:
This is essentially a makeshift DLL export check, the main thing I was checking for is “isPlextor” and “PreventLowSpeed”, with the reference to plextor.dll essentially being a “bonus”. I believe those exports to be specific to mcp.dll itself.

Reply:
Can you make that a "report on GitHub" then? I'm still hesitant.

Reply:
There’s nothing to report, that string is found in mcp.dll, of which I have several consistent samples.